src/popup/pages/AuthorizedAccounts.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <DetailsItem
+    :label="$t('pages.multisig-details.authorized-account')"
+  >
+    <template #value>
+      <div class="authorized-accounts">
+        <div class="left-side">
+          <div
+            v-for="address in accountsAddress"
+            :key="address"
+            class="account-row"
+          >
+            <Avatar
+              class="avatar"
+              size="sm"
+              :address="address"
+            />
+            <AddressTruncated
+              :address="address"
+              class="ae-address"
+            />
+            <LinkButton :to="getExplorerPath(address)">
+              <ExternalLink class="external-icon" />
+            </LinkButton>
+          </div>
+        </div>
+        <div class="right-side">
+          <div>
+            {{ $t('pages.multisig-details.signers') }}
+            <span class="bold">
+              {{ authorizedCountMock }}
+            </span>
+          </div>
+          <div>
+            {{ $t('pages.multisig-details.approval') }}
+            <span class="bold">
+              {{ needForApprovalCountMock }}
+            </span>
+          </div>
+        </div>
+      </div>
+    </template>
+  </DetailsItem>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from '@vue/composition-api';
+import Avatar from '../components/Avatar.vue';
+import AddressTruncated from '../components/AddressTruncated.vue';
+import ExternalLink from '../../icons/external-link.svg?vue-component';
+import { useGetter } from '../../composables/vuex';
+import LinkButton from '../components/LinkButton.vue';
+import DetailsItem from '../components/DetailsItem.vue';
+
+export default defineComponent({
+  name: 'AuthorizedAccounts',
+  components: {
+    DetailsItem,
+    LinkButton,
+    AddressTruncated,
+    Avatar,
+    ExternalLink,
+  },
+  props: {
+    accountsAddress: {
+      type: Array as PropType<string[]>,
+      default: () => [],
+    },
+  },
+  setup() {
+    const getExplorerPath = useGetter('getExplorerPath');
+
+    const authorizedCountMock = 3;
+    const needForApprovalCountMock = 3;
+
+    return {
+      getExplorerPath,
+      authorizedCountMock,
+      needForApprovalCountMock,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/variables';
+@use '../../styles/typography';
+
+.authorized-accounts {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+
+  .account-row {
+    display: flex;
+    align-items: center;
+  }
+
+  .avatar {
+    margin-right: 4px;
+  }
+
+  .external-link {
+    margin-left: 7px;
+    width: 22px;
+    height: 22px;
+
+    .external-icon {
+      opacity: 1;
+    }
+  }
+
+  .left-side,
+  .right-side {
+    width: 100%;
+  }
+
+  .left-side {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  .ae-address {
+    color: rgba(variables.$color-white, 0.85);
+  }
+
+  .right-side {
```

Comment:
for me `left-side` and `right-side` are not meaningful enough - they should be renamed